### DISCUSSION TITLE
Individual postgres instance security question

### DISCUSSION THREAD

**Discussion by osseonews** [nhost/nhost#1005] [Q&A]
Is there a way to restrict access to the individual postgres instance by IP address? We are used to using DO managed databases for our applications and DO has a section called: "Trusted Sources". Only IP addresses entered there can access the database. Is something like this available yet with Nhost individual postgres instance or is any client that has the necessary database credentials able to access the database? Thanks.

**Comment by elitan** [comment]
We don't have such a feature, but it's a good suggestion.

Similar issue: https://github.com/nhost/nhost/issues/221

**Reply by osseonews** [reply]
Thanks. I think this is critical for security. Please consider it.

**Reply by osseonews** [reply]
is there any update on this? How do we secure our postgres so it's only available to whitelisted IPs?

**Reply by elitan** [reply]
We'll update here as soon as there are any updates. This feature is on our roadmap but we don't have an ETA on when it's going to be available.
